### PR TITLE
Python3 and PyQt6 Support

### PR DESCRIPTION
**Pull request by javadr**
I have made several tweaks in compliance with Python3 and PyQt6. 
Now it works like a charm. ;D

### CODE REVIEW

File: negar/gui.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (shahinism) [inline negar/gui.py:1]: This discovery depends on the system installation. Nowadays most of the modern OSs are dropping version 2 family and `python` refers to version 3.

File: negar/gui.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
-from PyQt4.QtGui import *
-from PyQt4.QtCore import *
-from virastar import PersianEditor, add_to_untouchable
+from PyQt6.QtGui import *
+from PyQt6.QtCore import *
+from PyQt6.QtWidgets import *
+try:
+    from virastar import PersianEditor, add_to_untouchable
+except:
+    from .virastar import PersianEditor, add_to_untouchable
```
Comment (shahinism) [inline negar/gui.py:12]:
```suggestion
from negar.virastar import PersianEditor, add_to_untouchable
```

**Comment by shahinism** [comment]
Hi, thanks for your collaboration. 
However, the changeset is too big for a codebase that wasn't in active development for years. Can you please help with reviewing it, by adding a proper test suite proving the changes you are tending to apply?

File: negar/gui.py (hunk 2 above)
Comment (javadr) [reply negar/gui.py:12]: Actually, in my machine `from negar.viraster import ...` just loads from the installed library (`/lib/python3.10/site-packages/python_negar-0.6.6-py3.10.egg/negar`), not the local one and I made it to fix this issue! I meant when I wanted to run the `gui.py` from the `negar` directory it threw the following error:
`ModuleNotFoundError: No module named 'negar'`

**Comment by javadr** [comment]
BTW, I have some ideas to improve your great job and it would be my pleasure to help you in this manner. 
I used to use `virastar` but I have no idea what happened to that cool service as its web server is not accessible now.

File: negar/gui.py (hunk 1 above)
Comment (javadr) [reply negar/gui.py]: fixed.

**Comment by shahinism** [comment]
@javadr invited you as a collaborator to the project. Suppose you are interested; feel free to take ownership of it. Unfortunately, currently, I don't have an interest in maintaining it. So appreciate your initiative and would like to support it if you are open to it.

**Comment by javadr** [comment]
Thank you for the support. I will do my best to approach version 1.0.

**Pull request merged**